### ISSUE TITLE
Uninstalling iTorch

### ISSUE THREAD

**Issue by RishabhMalviya** [facebook/iTorch#163]
I followed the installation instructions given [here](https://gist.github.com/jetsonhacks/ad01566922a5c2648a10) for installing iTorch.

But I was getting way too many errors trying to run it, so I tried uninstalling it. I basically un-did the last few lines in the above-mentioned script:
```
luarocks install luacrypto 
luarocks install env
luarocks install lzmq
luarocks install lbase64
luarocks install uuid
# Install iTorch
git clone https://github.com/facebook/iTorch.git
cd iTorch
sudo env "PATH=$PATH" luarocks make
sudo chown -R $USER $(dirname $(ipython locate profile))
```
I deleted the cloned iTorch repo after installation. Then I uninstalled all the packages installed above with `luarocks remove`. 

**TL;DR: Basically, `iTorch` didn't uninstall as smoothly as expected, and I had to do some manual deletions, followed by `--force`ed removals of the other packages. How do I make luarocks forget about iTorch completely?**

After running `luarocks remove itorch`, I got this error:

`Error: Failed deleting /home/rishabh/torch/install/share/lua/5.1/itorch/gfx.lua`

so I deleted the directory `/home/rishabh/torch/install/share/lua/5.1/itorch` manually. 

Then, when I tried to delete all of the remaining dependencies (`luacrypto`, `env`, etc.), I had to do it with the `--force` flag because luarocks thought itorch was still installed, and it was checking dependency issues for it. I ended up with the following message after the final (`--force`ed) removal of the `uuid` package:
```
$ luarocks remove --force uuid

Removing uuid 0.2-1...
Missing dependency for itorch scm-1: lbase64 
Missing dependency for itorch scm-1: env 
Missing dependency for itorch scm-1: luacrypto 
Missing dependency for itorch scm-1: lzmq >= 0.4.2
Missing dependency for itorch scm-1: uuid 
Removal successful.
``` 

How do I make luarocks forget about iTorch completely?

**Comment by RishabhMalviya** [comment]
I think it may have to do with all of the `sudo`ing in the installation script at the end. I know it's said to use luarocks without sudo, but I really wasn't paying attention when running the script.
